gems/grpc/CVE-2023-32731.yml

```diff
@@ -0,0 +1,24 @@
+---
+gem: grpc
+cve: 2023-32731
+ghsa: cfgp-2977-2fmm
+url: https://github.com/grpc/grpc/pull/32309
+title: Connection confusion in gRPC
+date: 2023-07-05
+description: |
+  When gRPC HTTP2 stack raised a header size exceeded error, it skipped
+  parsing the rest of the HPACK frame. This caused any HPACK table mutations to also
+  be skipped, resulting in a desynchronization of HPACK tables between sender and
+  receiver. If leveraged, say, between a proxy and a backend, this could lead to requests
+  from the proxy being interpreted as containing headers from different proxy clients
+  - leading to an information leak that can be used for privilege escalation or data
+  exfiltration. We recommend upgrading beyond the commit contained in  https://github.com/grpc/grpc/pull/32309
+cvss_v3: 7.4
+patched_versions:
+  - ">= 1.53.0"
```

Comment:
The NVD entry says versions `>= 1.53.0, < 1.55.0` are effected. Not sure where they are getting `1.53.0` from, but I think patched versions should be `>= 1.55.0`.

Reply:
redo found that 1.53.1 was the right answer